### ISSUE TITLE
bindswitch doesn't run on initial sway startup

### ISSUE THREAD

**Issue by bionade24** [swaywm/sway#5814]
### Please fill out the following:
- **Sway Version:**
  sway version v1.5

- **Description:**
My current bindswitch config:
```
set $laptop eDP-1
bindswitch --reload --locked lid:on output $laptop disable
bindswitch --reload --locked lid:off output $laptop enable
```
bindswitch will obviously work on sway reload but not on the initial load, so that if the Laptop boots in the docking station, the display gets activated

Solution proposals:

1. Run bindswitch always on the initial config load
2. Add a `--load` flag to bindswitch to work on initial load

**Comment by user18130814200115-2** [comment]
I don't think this is a bug, bindswitch seems to activate as soon as you close the lid, in other words, it does not keep sending the signal and executing the command, that would be a waste of recources. When you start your laptop with the lid closed, sway recieves neither the lid:on nor the lid:off signal.
I propose you write a bash script that checks the status of the lid and run it on startup.

Here is a link to get you on your way:
https://unix.stackexchange.com/questions/148197/determine-status-of-laptop-lid

**Comment by MichaelMackus** [comment]
I had the same issue during my current sway setup. I thought `--reload` would handle this, but I guess not. Anyway, here's what I added to my sway config to get this working:

exec ( cat /proc/acpi/button/lid/LID/state | grep closed ) && swaymsg "output eDP-1 disable"
